src/Adyen/Util/Uuid.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen API Library for PHP
+ *
+ * Copyright (c) 2019 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+namespace Adyen\Util;
+
+/**
+ * Class Uuid
+ * @package Adyen\Util
+ */
+class Uuid
+{
+
+    /**
+     * Generate a UUID V4
+     *
+     * @return string
+     */
+    public static function generateV4()
+    {
+        $random = openssl_random_pseudo_bytes(32, $isSourceStrong);
```

Comment:
Isn't the $isSourceStrong missing?

Reply:
no see docs: If passed into the function, this will hold a boolean value that determines if the algorithm used was "cryptographically strong", e.g., safe for usage with GPG, passwords, etc. TRUE if it did, otherwise FALSE
https://www.php.net/manual/en/function.openssl-random-pseudo-bytes.php